File: mycuration/src/main/java/com/phicdy/mycuration/rss/CurationCondition.java
package com.phicdy.mycuration.rss;

public class CurationCondition {
	public static final String TABLE_NAME = "curationConditions";
	public static final String CURATION_ID = "curationId";
	public static final String WORD = "word";
	public static final String ID = "_id";

	public static final String CREATE_TABLE_SQL =
			"create table " + TABLE_NAME + "(" +
					ID + " integer primary key autoincrement,"+
					WORD + " text," +
					CURATION_ID + " integer," +
					"foreign key(" + CurationSelection.CURATION_ID + ") references " + Curation.TABLE_NAME + "(" + Curation.ID + "))";
}

File: domain/src/main/java/com/phicdy/mycuration/domain/entity/CurationCondition.kt
package com.phicdy.mycuration.domain.entity

object CurationCondition {
    const val TABLE_NAME = "curationConditions"
    const val CURATION_ID = "curationId"
    const val WORD = "word"
    private const val ID = "_id"

    const val CREATE_TABLE_SQL = "create table " + TABLE_NAME + "(" +
            ID + " integer primary key autoincrement," +
            WORD + " text," +
            CURATION_ID + " integer," +
            "foreign key(" + CurationSelection.CURATION_ID + ") references " + Curation.TABLE_NAME + "(" + Curation.ID + "))"
}
